Files/Views/MainPage.xaml.cs

```diff
@@ -354,12 +355,33 @@ private static async Task SetSelectedTabInfoAsync(TabItem selectedTabItem, strin
                     tabLocationHeader = currentPath.TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Split('\\', StringSplitOptions.RemoveEmptyEntries).Last();
                 }
             }
+
             if (tabHeader != null)
             {
                 tabLocationHeader = tabHeader;
             }
+
+            FilesystemResult<StorageFolderWithPath> currentItem = await FilesystemTasks.Wrap(() => DrivesManager.GetRootFromPathAsync(currentPath));
```

Comment:
Should this be put inside the `else` at line 352? Here it may overwrite the header set for special folders.